ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -282,7 +283,13 @@ - (void)updateCacheToItem:(ADTokenCacheItem *)cacheItem
     NSString* savedRefreshToken = cacheItem.refreshToken;
     if (isMRRT)
     {
-        AD_LOG_VERBOSE(correlationId, @"Token cache store - Storing multi-resource refresh token for authority: %@", _authority);
+        if ([ADAuthorityUtils isKnownHost:[_authority adUrl]])
```

Comment:
nit: I would simply do ternary operator here and just say "unknown host" for the latter or something